test/running_modules/run_tests.sh

```diff
@@ -55,3 +58,6 @@ should_fail run --module module/no_main_function
 
 # Main function with wrong arity
 should_fail run --module module/wrong_arity
+
+# try running gleam_module_javascript_test, which will crash if target is erlang
+should_succeed run --module gleam_module_javascript_test
```

Comment:
Can we have cases for explicitly setting the Erlang and JavaScript values too please?

Reply:
Added the tests:
```
should_succeed run --module gleam_module_javascript_test --target=javascript
should_fail run --module gleam_module_javascript_test --target=erlang
```